Transfer/Sources/TransferAmountCell.swift

```diff
@@ -85,23 +85,33 @@ extension TransferAmountCell: UITextFieldDelegate {
     func textField(_ textField: UITextField,
                    shouldChangeCharactersIn range: NSRange,
                    replacementString string: String) -> Bool {
+        let allowedDigitsCount = 14
         var currentText = textField.text ?? ""
         if string.count > 1 {
             // Paste
             let stringPastedAmount = string.format(with: detailTextLabel?.text)
+            let pastedDigitsOnly = stringPastedAmount.replacingOccurrences(of: "[^0-9]",
```

Comment:
why do this? If there's any invalid character other than digits then it shouldn't be allowed to paste

Reply:
That's what we want 123,467,543,123.99 to display on UI